samples/Mvc.Client/Startup.cs

```diff
@@ -51,6 +51,17 @@ public void ConfigureServices(IServiceCollection services)
                 options.ClientSecret = "qbxvkjk5la7mjp6";
             });
 
+            #region DisabledProvider
```

Comment:
Either remove the region and make it not commented out, or remove it entirely. We have over 50 providers now, so we can't have them all in sample project, so I'd probably err on the side of just removing it.

Reply:
fix in next commit